tests/helpers.py

```diff
@@ -12,11 +17,13 @@ def make_transfer_trace(
     to_address: str,
     token_address: str,
     amount: int,
+    action: dict = {},
```

Comment:
Scary python thing here

Using a mutable type (ex: dict, list) as a default shares the same memory across all default usages!

Example:
```
def myfunc(x = []):
    x.append(1)
    print(x)

myfunc()
myfunc()
```

prints
```
[1]
[1, 1]
```

The common pattern is to instead do:
```
def myfunc(maybe_x = None):
    x = [] if maybe_x is None else maybe_x
    x.append(1)
    print(x)
```

which now gives
```
[1]
[1]
```